packages/react/src/components/uiComponents.tsx

```diff
@@ -456,34 +483,86 @@ export const OrganizationSwitcher: OrganizationSwitcherExportType = Object.assig
   __experimental_Outlet: OrganizationSwitcherOutlet,
 });
 
-export const OrganizationList = withClerk(({ clerk, ...props }: WithClerkProp<OrganizationListProps>) => {
-  return (
-    <Portal
-      mount={clerk.mountOrganizationList}
-      unmount={clerk.unmountOrganizationList}
-      updateProps={(clerk as any).__unstable__updateProps}
-      props={props}
-    />
-  );
-}, 'OrganizationList');
+export const OrganizationList = withClerk(
+  ({ clerk, component, fallback, ...props }: WithClerkProp<OrganizationListProps & FallbackProp>) => {
+    const mountingStatus = useWaitForComponentMount(component);
+    const shouldShowFallback = mountingStatus === 'rendering' || !clerk.loaded;
 
-export const GoogleOneTap = withClerk(({ clerk, ...props }: WithClerkProp<GoogleOneTapProps>) => {
-  return (
-    <Portal
-      open={clerk.openGoogleOneTap}
-      close={clerk.closeGoogleOneTap}
-      props={props}
-    />
-  );
-}, 'GoogleOneTap');
+    const rendererRootProps = {
+      ...(shouldShowFallback && fallback && { style: { display: 'none' } }),
+    };
 
-export const Waitlist = withClerk(({ clerk, ...props }: WithClerkProp<WaitlistProps>) => {
-  return (
-    <Portal
-      mount={clerk.mountWaitlist}
-      unmount={clerk.unmountWaitlist}
-      updateProps={(clerk as any).__unstable__updateProps}
-      props={props}
-    />
-  );
-}, 'Waitlist');
+    return (
+      <>
+        {shouldShowFallback && fallback}
+        {clerk.loaded && (
+          <ClerkHostRenderer
+            component={component}
+            mount={clerk.mountOrganizationList}
+            unmount={clerk.unmountOrganizationList}
+            updateProps={(clerk as any).__unstable__updateProps}
+            props={props}
+            rootProps={rendererRootProps}
+          />
+        )}
+      </>
+    );
+  },
+  { component: 'OrganizationList', renderWhileLoading: true },
+);
+
+export const GoogleOneTap = withClerk(
+  ({ clerk, component, fallback, ...props }: WithClerkProp<GoogleOneTapProps & FallbackProp>) => {
+    const mountingStatus = useWaitForComponentMount(component);
+    const shouldShowFallback = mountingStatus === 'rendering' || !clerk.loaded;
+
+    const rendererRootProps = {
+      ...(shouldShowFallback && fallback && { style: { display: 'none' } }),
+    };
```

Comment:
`ClerkHostRenderer` always has to render, which is why we can't use Suspense, so to ensure the fallback and the component aren't mounted at the same time, we apply `display: none` to the renderer while the fallback is loading.